### PR TITLE
Add Gitlab LDAP EE only vars

### PR DESCRIPTION
**Pull request by BrutalBirdie**
Linked to issue #4

### CODE REVIEW

**Comment by RipJitter** [comment]
I tested this for you, but the variables did not show up with `gitlab_edition: gitlab-ee`. It seems like they only show with a `gitlab_version: gitlab-ee`. Setting `gitlab_version` to the aforementioned value causes the "Install GitLab" step to error out with `"No package gitlab-ce-gitlab-ee available."`

File: templates/gitlab.rb.j2 [hunk 1]
```diff
@@ -68,6 +68,11 @@ gitlab_rails['ldap_servers'] = YAML.load <<-EOS
     user_filter: '{{ values.user_filter | default(gitlab_ldap_user_filter) }}'
 
     attributes: {{ values.attributes | default(gitlab_ldap_attributes) }}
+{% if gitlab_version is defined and gitlab_version == "gitlab-ee" %}
```
Comment (RipJitter) [inline templates/gitlab.rb.j2]: I think if this was `gitlab_edition` instead of `gitlab_version` it would work.

Comment (BrutalBirdie) [reply templates/gitlab.rb.j2]: ah yes truly! I skipped into the wrong line in the README.
Silly me, fixing it now.

**Comment by RipJitter** [comment]
This seems to work after changing `gitlab_version` to `gitlab_edition`. (The variables are in the gitlab.rb, but I still am not seeing any administrative section appear with the admin group) I was getting timeouts during the install, but I see you have a handy `gitlab_install_timeout` for this. My only other concern is the varying templating of booleans. It seems sometimes they need to be strings in order for the casing to be correct. Would it be worth adding a "to_json" `{{ gitlab_XXXXXX | to_json }}` to the variables? This would allow boolean variables and keep the correct casing on the template. I have done this for a few PHP config files and it seems to work. Only a few weird strings like passwords needed to be manually wrapped in single quotes.

Example:
- `nginx['redirect_http_to_https'] = {{ gitlab_redirect_http_to_https | lower }}`
- `active_directory: {{ values.active_directory | default(gitlab_ldap_active_directory) }}`
Both are booleans, but handled in different ways. I think `to_json` could be used on almost everything to keep it uniform.

Up to you, just trying to think of improvements.

**Comment by BrutalBirdie** [comment]
> Would it be worth adding a "to_json" `{{ gitlab_XXXXXX | to_json }}` to the variables? This would allow boolean variables and keep the correct casing on the template. I have done this for a few PHP config files and it seems to work. Only a few weird strings like passwords needed to be manually wrapped in single quotes.
> 
> Example:
> 
>     * `nginx['redirect_http_to_https'] = {{ gitlab_redirect_http_to_https | lower }}`
> 
>     * `active_directory: {{ values.active_directory | default(gitlab_ldap_active_directory) }}`
>       Both are booleans, but handled in different ways. I think `to_json` could be used on almost everything to keep it uniform.
> 
> 
> Up to you, just trying to think of improvements.

This might be something good to try.
But lets do that in another PR to keep it simple.

**Pull request merged**